include/aws/common/error.h

```diff
@@ -16,8 +16,13 @@
  * permissions and limitations under the License.
  */
 
-#include <aws/common/common.h>
+#include <aws/common/assert.inl>
```

Comment:
why include this .inl directly? I believe the intent is to include the .h file, and have this come along for the ride.

Reply:
Done